source/js/text-ellipsis.js

```diff
@@ -1,14 +1,90 @@
-/*** 
- * @summary Sets the ellipsis based on the cpl - 3.
- * @param {Object} textLine: Node element, where to set the ellipsis
- * @param {Integer} characterPerLine: Characters per line. Set or calculate cpl.
+var cpl = require("./characters-per-line");
+var ellipsis = require("html-ellipsis");
+cpl.initFontConstant();
+
+var EllipsisUtil = {
+  /***
+   * @param {Object} lineStyles: Node with DOM elements that has a string
+   * @return {Boolean}
+   */
+  hasSpecificHeight: function(lineStyles) {
+    return lineStyles.height !== "auto" && parseInt(lineStyles.height) !== 0;
+  },
+
+  /***
+   * @summary Returns the width of the object
+   * @param {Node} textLines: Node with DOM elements that has a string
+   * @return {Integer} width
+   */
+  getWidth: function(textLine) {
```

Comment:
getWidth and getParent's width can be very specific preparations for getting a width from a very specific element. They shouldn't be in the text-ellipsis.js, because next time it's a very specific different element, where we need a width from.
So what is your idea to place it here?

Reply:
Well, i see no more use for it besides helping to calculate the Ellipsis (for now), so i put it on the Ellipse utilities ... I don't know where else should i put it..